src/style.css

```diff
@@ -1,3 +1,61 @@
 body {
   margin: 0;
+  background-color: #f4f4f4;
+  font-family: Roboto, Arial, sans-serif;
+
+  --accent-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background-color: #fff;
+}
+
+.logo-link {
+  display: flex;
+}
+
+.nav-links {
+  list-style: none;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-links .nav-link-item:not(:last-child) {
+  margin-right: 20px;
```

Comment:
It is recommended to be consistent with your vertical margins. In this case, you are applying a margin to the right of each item. Make sure you follow the same approach in other parts of your code. If there's a need to add vertical margins, it's better to stick with either top or bottom, but not both.